src/app/features/issue/providers/caldav/caldav-client.service.ts

```diff
@@ -402,6 +402,14 @@ export class CaldavClientService {
     }
     todo.updatePropertyWithValue('last-modified', now);
     todo.updatePropertyWithValue('dtstamp', now);
+    
```

Comment:
Could you maybe also add a comment on why we need this?

Reply:
Done